### PR TITLE
Add SVG Cell in Table Component

### PR DESCRIPTION
**Pull request by miguelangellv**
**What this PR does / why we need it**:

That PR add SVG Image Cell to Table Component. Now, you can add SVG images, as Material Icons, with thresholds color and  value mapping support.

Is very usefull for show states (online/offline for example).

**Which issue(s) this PR fixes**:

Support SVG imgaes

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by torkelo** [comment]
Why does this need new cell? Can't it use the image cell?

**Comment by MiguelAngelLV** [comment]
@torkelo 

Image cell use URL to image.  SVG cell use the SVG directly.

With SVG, the SVG color change using thresholds...

And not use  tag, use <svg> tag.

Could they be merged with a fews checks as "start with <svg" but... I think is more clean two cells.

File: packages/grafana-ui/src/components/Table/SVGCell.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+  const displayValue = field.display!(cell.value);
+
+  const cellStyle = getCellStyle(tableStyles, field, displayValue);
+
+  return (
+    <div
+      {...cellProps}
+      className={cellStyle + ' ' + tableStyles.imageCell}
+      dangerouslySetInnerHTML={{ __html: displayValue.text }}
```
Comment (torkelo) [inline packages/grafana-ui/src/components/Table/SVGCell.tsx:17]: we cannot allow this, not secure

**Comment by torkelo** [comment]
> With SVG, the SVG color change using thresholds...

I do not see anything here that relates to thresholds

File: packages/grafana-ui/src/components/Table/SVGCell.tsx (hunk 1 above)
Comment (MiguelAngelLV) [reply packages/grafana-ui/src/components/Table/SVGCell.tsx:17]: I could add safe-html (https://www.npmjs.com/package/safe-html) to ensure that is only a SVG.

Ok?

**Comment by ryantxu** [comment]
Would it make more sense to have an HTML cell type?  and then manage the values the same way we do for the text panel?  https://github.com/grafana/grafana/blob/master/public/app/plugins/panel/text/TextPanel.tsx#L76

**Comment by MiguelAngelLV** [comment]
Welll... Now it works as "Html Cell"..  I can adapt it...

**Comment by MiguelAngelLV** [comment]
@ryankurte TextPanel only works with markdown... And I don't know if markdown support svg

File: packages/grafana-ui/src/components/Table/SVGCell.tsx (hunk 1 above)
Comment (torkelo) [reply packages/grafana-ui/src/components/Table/SVGCell.tsx:17]: we already have sanitize functions in grafana/data https://github.com/grafana/grafana/blob/master/packages/grafana-data/src/text/sanitize.ts

Comment (MiguelAngelLV) [reply packages/grafana-ui/src/components/Table/SVGCell.tsx:17]: Jums... I see that configuration is only for CSS attributes. So, I need create a new config por XSS with some tags.

I think... a, span, strong, italic, svg, img, h1-6.... any else?

**Comment by dprokop** [comment]
@MiguelAngelLV thank you for the contribution! I don't think we need a special cell type to display SVGs. You can already do that using base64 encoded SVG and image cells.

**Comment by MiguelAngelLV** [comment]
@dprokop the problem with base64 is that you can't change the color dynamically

**Comment by peterholmberg** [comment]
Hi @MiguelAngelLV, we don't want to add more table cell types at this point without a strong need from the community, therefore we have decided to close this PR for now and encourage you to open a [feature request](https://github.com/grafana/grafana/issues/new?labels=type%3A+feature+request&template=2-feature_request.md) for this. If it gains traction and interest from the community we can reopen this pull request.

**Pull request closed without merging**